about.html

```diff
@@ -0,0 +1,117 @@
+                <li><a href="index.html" > Home</a></li>
+                <li> <a href="#"> Live_Streaming </a></li>
+                <li class="disapper"> <a href="index.html#event-categories"> Event </a></li>
+                <li class="disapper"><a  href="#event">See recent</a></li>
+                <li class="disapper"> <a href="#">  Logout </a></li>
+            </ul>
+        </nav>
+    </header>
+<main>
+    <section class="descrption">
+    <h2>GLOBAL . STREAMING</h2>
+    <h2> BEST LIVE STREAMING WEBSITE -2022</h2>
+    <div>
+        <p>Global Streaming is a live-streaming website that allow users to record, 
+            broadcast video and stream it live simultaneously. Also it allows users to broadcast
+             live recordings of videos and audio. Others who have access to the Global Live video streaming,
+              have fueled influencer marketing, entertainment, business marketing, and the way people interact 
+              over the internet.
+             Global Streaming is the best website for sharing and streaming live event in 2022.</p>
+    </div>
+    <p>Please contact us via email for feedback or any other inquiry about Global Streaming</p>
+    <i>globalstreaming2022@gmail.com</i>
+    </section>
+    <section class="recent-events" id="event">
+     <h5>See recent streamed events</h5>
+     <hr>
+     <p> Take a look at three top past events which was streamed live here .....</p>
+     <div class="embeded-video">
+        <div class="video">
+          <iframe width="100%" height="270" src="https://www.youtube.com/embed/c78H-nolqxI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+        </div>
+        <div class="video">
+          <iframe width="100%" height="270" src="https://www.youtube.com/embed/A_9m5INlV-M" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+        </div>
+        <div class="video">
+          <iframe width="100%" height="270" src="https://www.youtube.com/embed/7WU__JQuB8g" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+        </div>
+      </div>
+    </section>
+    <section class="other-categories">
+        <h4> Follow Us </h4>
+        <hr>
+        <ul>
+            <li> <span class="fa-brands fa-instagram"></span> Instagram</li> 
+            <li class="disable"><span class="fa-brands fa-twitter"></span> Twitter</li>
+            <li><span class="fa-brands fa-facebook"></span> Facebook</li>
+            <li class="disable"><span class="fa-brands fa-linkedin"></span> Linkedin</li>
+            <li><span class="fa-brands fa-telegram"></span> Telegram</li>
+        </ul> 
+      </section>
```

Comment:
- [ ] Please know that you shouldn't change this section from the one on the home page, they should both have the same content, and also this section should be only visible on mobile and it should be hidden on desktop, kindly fix this.